docs/animations/FadeHeader.md

```diff
@@ -8,101 +8,72 @@ keywords: windows 10, uwp, uwp community toolkit, uwp toolkit, fadeheader, fadeh
+</ListView>
 ```
 
-
-**Explicit usage**: 
-
-Set the ElementName property using the UIElement of the Header manually
-
+***Explicit usage***: Set the ElementName property using the UIElement of the Header manually
 ```xml
+<Page ...
+    xmlns:interactivity="using:Microsoft.Xaml.Interactivity"  
+    xmlns:behaviors="using:Microsoft.Toolkit.Uwp.UI.Animations.Behaviors"/>
 
+<ListView>
     <interactivity:Interaction.Behaviors>
         <behaviors:FadeHeaderBehavior HeaderElement="{Binding ElementName=MyHeaderGrid}" />
     </interactivity:Interaction.Behaviors>
-
+    <ListView.Header>
+        <Grid Name="MyHeaderGrid">
+            <!-- Header Content -->
+        </Grid>
+    </ListView.Header>
+</ListView>
 ```
 
-### C# Implementation ###
-
-Implicit usage:
+**C#**
 
+***Implicit usage***:
 ```csharp
-
-    Microsoft.Xaml.Interactivity.Interaction.GetBehaviors(MyListView).Add(new FadeHeaderBehavior());
-
+Microsoft.Xaml.Interactivity.Interaction.GetBehaviors(MyListView).Add(new FadeHeaderBehavior());
 ```
 
-Explicit usage:
-
+***Explicit usage***:
 ```csharp
-
-    Microsoft.Xaml.Interactivity.Interaction.GetBehaviors(MyListView).Add(new FadeHeaderBehavior { HeaderElement = MyHeaderGrid });
-
-```
-
-
-## Example ##
-
-```xml
-
-    <ListView x:Name="MyListView">
-    <interactivity:Interaction.Behaviors>
-        <behaviors:FadeHeaderBehavior />
-    </interactivity:Interaction.Behaviors>
-    <ListView.Header>
-        <Grid x:Name="MyHeaderGrid"
-                      MinHeight="250"
-                      Background="{StaticResource Brush-Blue-01}">
-            <StackPanel VerticalAlignment="Center"
-                        HorizontalAlignment="Center"
-                        Margin="20,0">
-                <TextBlock Text="This Is The Header"
-                           TextAlignment="Center"
-                           FontWeight="Bold"
-                           Style="{ThemeResource TitleTextBlockStyle}"
-                           Foreground="{StaticResource Brush-White}"
-                           Margin="0,5" />
-                <TextBlock Text="It starts with 100% opacity but will fade to 0% as you scroll up."
-                           Style="{ThemeResource SubtitleTextBlockStyle}"
-                           Foreground="{StaticResource Brush-White}"
-                           VerticalAlignment="Center"
-                           TextAlignment="Center" />
-            </StackPanel>
-        </Grid>
-    </ListView.Header>
-</ListView>
-
+Microsoft.Xaml.Interactivity.Interaction.GetBehaviors(MyListView).Add(new FadeHeaderBehavior { HeaderElement = MyHeaderGrid });
 ```
 
+## Sample Output
 
-[FadeHeader Behavior Sample Page Source](https://github.com/Microsoft/UWPCommunityToolkit/tree/master/Microsoft.Toolkit.Uwp.SampleApp/SamplePages/FadeHeader)
+![FadeHeader Behavior animation](https://github.com/Vijay-Nirmal/UWPCommunityToolkit/blob/DocImprovements/docs/resources/images/Animations/FadeHeader/Sample-Output.gif)
```

Comment:
This should refer to a toolkit repo resource not an external repo (https://github.com/Vijay-Nirmal)

Reply:
I thought it will automatically change after being merged

Reply:
Unfortunately no.

Reply:
Should I use relative link like ../resources/images/Animations/FadeHeader/Sample-Output.gif or direct reference link like https://github.com/Microsoft/UWPCommunityToolkit/blob/DocImprovements/docs/resources/images/Animations/FadeHeader/Sample-Output.gif

Reply:
Always use relative paths as the documentation is not hosted on github

Reply:
Done